src/config/dynamic-theme-fixes.config

```diff
@@ -1347,6 +1347,10 @@ span.loader-wrapper a,
 code:first-of-type {
     background-image: linear-gradient(to right, rgba(255,255,255,0.03),rgba(255,255,255,0.03)calc(1ch + 16px),transparent calc(1ch + 16px),transparent) !important;
 }
+.sc-1jjaulc-0 {
```

Comment:
Hmm, this is using randomized selectors(which tends to change every few days/weeks on sites using them), can this be changed to not rely on the randomized selectors?

Reply:
We talked about this in #5953. Jira isn't cooperative with this unfortunately.

Reply:
> We talked about this in #5953. Jira isn't cooperative with this unfortunately.

Woops. That's correct.